testing-plugin/src/test/kotlin/arrow/meta/plugin/testing/plugins/transform/TransformRemovePlugin.kt

```diff
@@ -0,0 +1,49 @@
+package arrow.meta.plugin.testing.plugins.transform
+
+import arrow.meta.Meta
+import arrow.meta.Plugin
+import arrow.meta.invoke
+import arrow.meta.quotes.Transform
+import arrow.meta.quotes.namedFunction
+
+val Meta.transformRemove: List<Plugin>
+  get() = listOf(transformRemoveSingleElement, transformRemoveSingleElementFromContext, transformRemoveElementsFromContext)
+
+private val Meta.transformRemoveSingleElement: Plugin
+  get() =
+    "Transform Remove" {
+      meta(
+        namedFunction({ name == "transformRemove" }) { f ->
+            Transform.remove(f)
+        }
+      )
+    }
+
+private val Meta.transformRemoveSingleElementFromContext: Plugin
+  get() =
+    "Transform Remove" {
```

Comment:
Let's name these "Remove Single Element From Context" and so on for more descriptive testing!